Makefile

```diff
@@ -213,8 +213,8 @@ TEST_PACKAGES = \
 	testing \
```

Comment:
You should be able to add `net/mail` here too.

Reply:
Thanks! Will do.